packages/project-access/test/project/cap.test.ts

```diff
@@ -88,6 +89,14 @@ describe('Test isCapJavaProject()', () => {
         ).toBeFalsy();
     });
 });
+describe('Test checkPathForCapApp()', () => {
```

Comment:
You could add a couple more tests here, one where cap project is the root, where the cap root is in the sub dirs, and then a negative test

Reply:
Thank you for the review. Added some.